### PR TITLE
templatize flink parallelism and kafka producer configurations

### PR DESCRIPTION
**Pull request by ravisingal**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -79,6 +81,9 @@ rawSpansGrouperConfig:
     sink:
       kafka:
         bootstrapServers: "bootstrap:9092"
+        lingerMs: 0
+        batchSize: 16384
+        compressionType: none
```
Comment (surajpuvvada) [inline helm/values.yaml:86]: we have overrides in prod for these am assuming ?

**Review by surajpuvvada — DISMISSED** [review]

File: helm/values.yaml (hunk 1 above)
Comment (ravisingal) [reply helm/values.yaml:86]: yes. will raise a PR in helm-values repo.

File: helm/templates/raw-spans-grouper-config.yaml [hunk 2]
```diff
@@ -8,12 +8,15 @@ data:
   application.conf: |-
     flink.job {
       {{- if hasKey .Values.rawSpansGrouperConfig.flink "job" }}
+      parallelism = {{ int .Values.rawSpansGrouperConfig.flink.job.parallelism }}
```
Comment (buchi-busireddy) [inline helm/templates/raw-spans-grouper-config.yaml:12]: @ravisingal Should we make this optional instead? Flink defaults to the no. of processors available I think.

Comment (ravisingal) [reply helm/templates/raw-spans-grouper-config.yaml:12]: done.

**Review by buchi-busireddy — APPROVED** [review]

**Pull request merged**